astropy/utils/iers/iers.py

```diff
@@ -191,7 +191,8 @@ class Conf(_config.ConfigNamespace):
         ["error", "warn", "ignore"],
         "IERS behavior if the range of available IERS data does not "
         "cover the times when converting time scales, potentially leading "
-        "to degraded accuracy.",
+        "to degraded accuracy.  Applies only to when using only IERS-B data.",
+          iers_degraded_accuracy:
```

Comment:
Something went wrong here - this is causing all the failures.

Reply:
Oops, a bad edit!